lib/team_api/cross_referencer.rb

```diff
@@ -41,6 +45,15 @@ def self.create_xref_data(site)
     end
     private_class_method :create_xref_data
 
+    def self.create_name_xrefs(team)
+      name_xrefs = {}
+      team.data.each do |member|
+        name_xrefs[member.last['deprecated_name']] =
+            member.last['name'] if member.last['deprecated_name']
```

Comment:
No, it shouldn't need to do any case correction. This is all coming from existing data, so it should already have the correct cases (which also all happen to be all-lowercase).